CLAUDE.md

```
@@ -0,0 +1,44 @@
+- **Playwright/**: End-to-end tests using Playwright
+- **WebAPp/**: Frontend application (Nuxt.js/Vue.js)
+
+## Database Setup
+
+To set up the database:
+
+1. Build the solution: `dotnet build Pdc.sln`
+2. Create local tool manifest (once): `dotnet new tool-manifest`
+3. Install EF tools locally: `dotnet tool install dotnet-ef`
+4. Create migration: `dotnet ef migrations add InitialCreate --startup-project Pdc.WebAPI --project Pdc.Infrastructure`
+5. Update database: `dotnet ef database update --startup-project Pdc.WebAPI --project Pdc.Infrastructure`
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines

(MD032, blanks-around-lists)